metpy/plots/declarative.py

```diff
@@ -657,29 +660,21 @@ def draw(self):
                 self.ax.set_extent(area, ccrs.PlateCarree())
 
             # Use the set title or generate one.
-            title = self.title or ', '.join(plot.name for plot in self.plots)
+            title = self.title or ', \n'.join(plot.name for plot in self.plots)
             self.ax.set_title(title)
             self._need_redraw = False
 
 
 @exporter.export
-class Plot2D(HasTraits):
-    """Represent plots of 2D data."""
+class Plots(HasTraits):
+    """Represent plots of 2D scalar data."""
```

Comment:
This is the same docstring as `Plot2D`. I feel like they should have something that differentiates them.

Reply:
Agreed - missed that!